.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: Create a report to help us improve 
+title: "🪲[BUG] <title>"
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the sections below to help everyone identify and fix the bug
+
+        **Contribution Guidelines**
+        - If you want to work on an issue, please let us know in a comment below.
+        - Do not begin work on the issue unless it has been assigned to you.
+        - If you do not respond in 3 days, the issue may be reassigned to someone else.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      placeholder: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce
+      placeholder: |
+        1. Go to page '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      placeholder: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Put here any screenshots or videos (optional)
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Extra fields
+      options:
+        - label: I have used the search function to check if an issue already exists
+          required: true
+        - label: I'd like to work on this issue
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting this bug! We will get back to you as soon as possible.
```

Comment:
Will this be added to the the final issue body as well?

Reply:
@sarthakroy107 ?

Reply:
I have removed it

Reply:
I wanted to ask if this field and the checkboxes will be displayed in the issue body. Will they?

Reply:
I apologise for my miss understanding
1- Last line can only be seen while creating the issue not in main body, nothing need to be changed.
2- Check boxes can be seen in main body.

![Screenshot_20231130_233600](https://github.com/kossiitkgp/KWoC-Frontend/assets/95567641/381ff504-f2dc-4998-8e7e-f1de6205651f)

Reply:
Then I think we can keep the last line and the checkboxes in all but the good first issue templates :)

Reply:
Understood